package-lock.json

```diff
@@ -1,8 +1,9503 @@
 {
   "name": "http-status-codes",
   "version": "2.2.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Can do!

Reply:
```suggestion
  "lockfileVersion": 1,
```

Reply:
Sorry for the confusion. I meant just reverting the entire `package.lock` file. I don't think it needs to be included here. 😅

Reply:
Righto, can do. Out of curiosity, any particular reason to not update the lock? And if so, maybe worth updating the .gitignore?

Reply:
I guess I would only expect the lock to change if the dependencies change, which they haven't in this PR.

Reply:
@prettymuchbryce reverted and is ready for merge.

I would only add that due to the generated nature of the lock file, it should be kept up to date as much as possible - as when dependencies DO change it would be loads easier to read the diff as the maintainer.